### PR TITLE
TDI-37699:change the version name to fit the wizard and migration task.

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-37699

### CODE REVIEW

**Comment by pyzhou-talend** [comment]
I change the DB version from Sybase_12 to Sybase_12_15 only to fit the wizard. "Sybase_12_15" still representative Sybase 12 . However I have to keep the name same with other Sybase components, since others doesn't have a singular Sybase 15 version.

File: main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQBulkExec/tSybaseIQBulkExec_java.xml [hunk 1]
```diff
@@ -642,9 +642,9 @@
 
    <CODEGENERATION>
     <IMPORTS>
-       <IMPORT NAME="Driver-Sybase3" MODULE="jconn3.jar" MVN="mvn:org.talend.libraries/jconn3/6.0.0"  BundleID="" REQUIRED_IF = "(DB_VERSION == 'SYBSEIQ_12') AND (USE_EXISTING_CONNECTION == 'false')" />
+       <IMPORT NAME="Driver-Sybase3" MODULE="jconn3.jar" MVN="mvn:org.talend.libraries/jconn3/6.0.0"  BundleID="" REQUIRED_IF = "(DB_VERSION != 'SYBSEIQ_16') AND (USE_EXISTING_CONNECTION == 'false')" />
```
Comment (nrousseau) [inline main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQBulkExec/tSybaseIQBulkExec_java.xml]: why != SYBSIQ_16??
do we need this jar for both SYBSEIQ_12_15 / SYBSEIQ_15 ? seems not the same as before..

File: main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQOutputBulkExec/tSybaseIQOutputBulkExec_java.xml [hunk 2]
```diff
@@ -331,9 +331,9 @@
 
     </TEMPLATES>
     <IMPORTS>
-       <IMPORT NAME="Driver-Sybase3" MODULE="jconn3.jar" MVN="mvn:org.talend.libraries/jconn3/6.0.0"  BundleID="" REQUIRED_IF = "(DB_VERSION == 'SYBSEIQ_12') AND (USE_EXISTING_CONNECTION == 'false')" />
+       <IMPORT NAME="Driver-Sybase3" MODULE="jconn3.jar" MVN="mvn:org.talend.libraries/jconn3/6.0.0"  BundleID="" REQUIRED_IF = "(DB_VERSION != 'SYBSEIQ_16') AND (USE_EXISTING_CONNECTION == 'false')" />
```
Comment (nrousseau) [inline main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQOutputBulkExec/tSybaseIQOutputBulkExec_java.xml]: same as the other component..

File: main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQBulkExec/tSybaseIQBulkExec_java.xml (hunk 1 above)
Comment (pyzhou-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tSybaseIQBulkExec/tSybaseIQBulkExec_java.xml]: Yes. We need jconn3.jar for both Sybase 12 and 15. I made a mistake on last version. And I correct it on this PR.

**Pull request merged**